README.md

```diff
@@ -0,0 +1,76 @@
+cerner_splunk_ingredient Cookbook
+=================================
+Resource cookbook which provides custom resources for installing Splunk.
+
+These resources can:
+- Install Splunk via downloadable package or archive
+
+Requirements
+------------
+Chef >= 12.4
+Ruby >= 2.2.3
```

Comment:
need to adjust this after our ruby version discussion earlier on this PR. Since chef 12.4.3 packages ruby 2.1.<something> we can't say we require 2.2.3 or above.